test/JDBC/expected/declare_atatglobalvars_upgrade-before-16_5-vu-verify.out

```diff
@@ -0,0 +1,20 @@
+insert into t1_declare_atatglobalvars_upgrade values(4)
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Incorrect syntax near '@@max_precision'.)~~
+
+exec p1_declare_atatglobalvars_upgrade
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Incorrect syntax near '@@max_precision'.)~~
```

Comment:
So we have a case where previous functions will stop working are we expecting the customer experience to be the same?
Can we take the risk of assuming that there wouldnt be such dependent objects in the first place since these are not valid TSQL?

Reply:
If such objects had existed, which would be highly unlikely since they could not have been created in SQL Server in the first place, then they would have produced incorrect results (see PR description). That was wrong, and this fix stops it from happening. This is an extreme edge scenario and would only occur if customers had created an object using syntax that was invalid in SQL Server.